### ISSUE TITLE
Is it possible to return the invalidated Table names?

### ISSUE THREAD

**Issue by marcellelek** [VahidN/EFCoreSecondLevelCacheInterceptor#244]
# Summary of the issue
We have 2 server the only common point is the dbcontext. If we invalidate the data on server A, the server B doesnt know about it.
If you could return the invalidated tablenames, than we could invalidate on server B.
Is there any code realted to this issue or any idea how to do it?

## Environment

```
.NET (Core) SDK version: 
Microsoft.EntityFrameworkCore version: 
EFCoreSecondLevelCacheInterceptor version: 
Database: (e.g. SQL Server, SQLite, etc.)
Cache provider: (e.g. in-memory, redis, etc.)
```

## Example code/Steps to reproduce:

```
paste your core code
```

## Output:

```
Exception message:
Full Stack trace:
```

**Comment by marcellelek** [comment]
Yes, but how can i invalidate if I dont invalidate by myself. It is done via your package. I want to know that in the query which tables are part of the query and then i want to send it to the server B. if that makes sense. I know i can manually invalidate. but in our project we dont do it manually.

Best regards.

**Comment by VahidN** [comment]
You should invalidate it manually in your scenario. You should inject the `IEFCacheServiceProvider` service somewhere in your code and then call its `InvalidateCacheDependencies` method yourself.
